src/core/mustache.rs

```diff
@@ -48,6 +48,21 @@ impl Mustache {
         }
     }
 
+    pub fn render_http(&self, value: &impl RequestString, method: &reqwest::Method) -> String {
```

Comment:
mustache doesn't care about HTTP. We shouldn't have this a dependency to this function. Mustache doesn't even care it being a URL.